apps/studio/schemaTypes/documents/post.ts

```diff
@@ -220,29 +219,28 @@ export const post = defineType({
       description:
         'This will be used for article snippets in social media and Google searches. Ideally between 140 and 160 characters.',
       group: GROUP.MAIN_CONTENT,
-      validation: (rule) => [
-        rule.required(),
+      validation: (rule) =>
         rule
+          .required()
           .min(140)
           .warning(
             'Excerpt should be at least 140 characters for optimal SEO visibility in search results.',
-          ),
-        rule
+          )
           .max(160)
           .warning(
-            'Excerpt should not exceed 160 characters as it will be truncated in search results.',
+            'Excerpt should be at most 160 characters for optimal SEO visibility in search results.',
           ),
```

Comment:
So this becomes `rule.required().min(140).warning('').max(160).warning('')` because if you have it in an array, it won't catch it with how we are catching the input validations for character counting and showing a max number.